controllers/songsController.js

```diff
@@ -0,0 +1,50 @@
+const express = require('express');
+const { getAllSongs, addNewSongs, getSong, deleteSong, updateSong } = require('../queries/song');
+const songs = express.Router();
+
+// const favSongs = require('../models/songs');
+
+songs.get('/', async (req, res) => {
+    const songs = await getAllSongs();
+    console.log(songs)
+    res.status(200).json(songs)
+});
+
+songs.get("/", async (req, res) => {
```

Comment:
You have two routes doing the same thing (line 7) one of these need to be deleted.